packages/clay-dropdown/src/ClayDropdownBase.soy

```diff
@@ -257,7 +257,11 @@
 
 	{call ClayPortal.render}
 		{param content kind="html"}
-			{$dropdownMenu}
+			{if $expanded}
```

Comment:
hey @travisrcory, this sounds like a good idea. But I was curious when you refer to performance, what performance do you mean? The first render?

Reply:
Hey @matuzalemsteles sorry for the confusion, yeah I was referring to the first render! Thanks for clarifying.

Reply:
Np, thanks!

Reply:
Just leaving one thought of me about this change:

- **Always rendered** - As soy takes care of doing the diff, the first rendering may be slow but the next interactions will be fast since the only change is just add to `show` for example.
- **Destroy every time** - When rendered only when it is expanded, the first rendering will be fast, but the subsequent interactions will be slow since we are adding and destroying every time it has an `expanded` change.

We may be trading for each other, I like the idea of ​​getting the first fast render, but I'd also like to get the subsequent quick interactions.

One thought of getting around this would be something like `lazy`, not rendering on the first render, rendering on the first interaction with the dropdown, and subsequent interactions just kept changing what is needed... We could say that this would be a "dynamic loading" triggered by interaction.

---

I noticed something in the tests with this change, when I'm with the scroll zero and open the dropdown it skips scroll, I can not say why...